tslint.json

```diff
@@ -4,6 +4,7 @@
         "max-line-length": false,
         "no-empty": false,
         "no-shadowed-variable": false,
-        "object-literal-sort-keys": false
+        "object-literal-sort-keys": false,
+        "cyclomatic-complexity": false
```

Comment:
You should try to resolve the cyclomatic complexity by refactoring the code rather than disabling the rule. If the refactoring takes too much effort and you'd rather tackle it separately, then you can disable the rule locally for the file only rather than disabling it globally for all files.

Reply:
Fixed!